_episodes/01-introduction.md

```diff
@@ -40,24 +40,24 @@ We wish to create a list of values, which of these lines of code is valid to do
 {: .solution}
 
 ### 3a. Indexing characters in a string
-Lists and strings both contain multiple indexed values (in the case of strings these are specifically individual characters rather than other values). If we have a variable `variable` which contains the string `value`, which of these options will print the first character (`v`) for us?
-1. `print(variable[0])`
-2. `print(variable[1])`
-2. `print(variable['v'])`
+Lists and strings both contain multiple indexed values (in the case of strings these are specifically individual characters rather than other values). If we have a variable `animal` which contains the string `penguin`, which of these options will print the first character (`p`) for us?
+1. `print(animal[0])`
+2. `print(animal[1])`
+2. `print(animal['p'])`
 
 > ## Solution
 > Answer 1 is correct
 {: .solution}
 
 
 ### 3b. Indexing characters in a string (slicing)
-We can also select whole sections of lists and strings, not just single elements. Using the same variable `variable`, containing the string `value`, as above, which of these options will print the last 2 characters for us? (Note that there is more an one correct answer)
-1. `print(variable[3:])`
-2. `print(variable[4:])`
-2. `print(variable[4:5])`
-3. `print(variable[3:5])`
-4. `print(variable[-2:])`
-5. `print(variable[:-2])`
+We can also select whole sections of lists and strings, not just single elements. Using the same variable `animal`, containing the string `penguin`, as above, which of these options will print the last 2 characters for us? (Note that there is more an one correct answer)
+1. `print(animal[3:])`
+2. `print(animal[4:])`
+2. `print(animal[4:5])`
+3. `print(animal[3:5])`
```

Comment:
This does show why `-2` is the more robust solution ;-)
Can you change to these slices:
1. [5:]
2. [6:]
3. [6:7]
4. [5:7]